### DISCUSSION TITLE
Can I use WebDAV / ftp server as plot storage?

### DISCUSSION THREAD

**Discussion by ArrayZoneYour** [Chia-Network/chia-blockchain#2012] [Q&A] [answered]
I store the plots on my NAS server and use another machine under the same local network to farm by setting remote shared folder on NAS as the plot directory. I don't know whether the remote shared folder read performance is enough to complete the POS/VDF.

**Comment by wjblanke** [comment, marked as answer]
We don't recommend it. Better to run with local drives or USB.

You are the only person that can test to see if it is working fast enough.

**Comment by erdincsk** [comment]
> I store the plots on my NAS server and use another machine under the same local network to farm by setting remote shared folder on NAS as the plot directory. I don't know whether the remote shared folder read performance is enough to complete the POS/VDF.

What was your results regarding that hmm dirty solution? did it work well?
